extensions/wikia/WikiFactory/SpecialWikiFactory_body.php

```diff
@@ -226,22 +271,17 @@ private function doWikiSelector() {
 	 *
 	 * show wiki data
 	 *
-	 * @access public
-	 * @author Krzysztof Krzyżaniak <eloy@wikia.com>
-	 *
-	 * @return nothing
 	 */
 	public function doWikiForm() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this function "doWikiForm" is 34 which is greater than 20 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1541)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This function "doWikiForm" has 200 lines, which is greater than the 150 lines authorized. Split it into smaller functions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS138)